ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/SyncTargetManager.java

```diff
@@ -74,7 +74,8 @@ public SyncTargetManager(
   protected CompletableFuture<Optional<EthPeer>> selectBestAvailableSyncTarget() {
     final BlockHeader pivotBlockHeader = fastSyncState.getPivotBlockHeader().get();
     final EthPeers ethPeers = ethContext.getEthPeers();
-    final Optional<EthPeer> maybeBestPeer = ethPeers.bestPeerWithHeightEstimate();
+    // just use the default (merge) comparator
+    final Optional<EthPeer> maybeBestPeer = ethPeers.bestPeer();
```

Comment:
I think this might be a step back thinking about PoS networks, no much use for the comparison using TTD since this is basically a constant after chain switches to PoS.

Reply:
I think it actually evaluates to the same thing in the end so I'm reverting this line change.